tests/api/acquisition/test_acquisition_workflow.py

```diff
@@ -0,0 +1,806 @@
+        'library': {'$ref': get_ref_for_pid('lib', lib_martigny.pid)}
+    }
+    martigny_root_account = _make_resource(
+        client, 'acac', martigny_root_account_data)
+
+    # STEP 2 :: Create child accounts for the martigny root account
+    martigny_child_books_account_data = {
+        'name': 'Martigny Books chid account',
+        'number': 'MTY.000b.00',
+        'allocated_amount': 2000,
+        'budget': {'$ref': get_ref_for_pid('budg', budget_2020_martigny.pid)},
+        'library': {'$ref': get_ref_for_pid('lib', lib_martigny.pid)},
+        'parent': {'$ref': get_ref_for_pid('acac', martigny_root_account.pid)},
+    }
+    martigny_books_account = _make_resource(
+        client, 'acac', martigny_child_books_account_data)
+
+    martigny_child_serials_account_data = {
+        'name': 'Martigny Serials chid account',
+        'number': 'MTY.000s.00',
+        'allocated_amount': 3000,
+        'budget': {'$ref': get_ref_for_pid('budg', budget_2020_martigny.pid)},
+        'library': {'$ref': get_ref_for_pid('lib', lib_martigny.pid)},
+        'parent': {'$ref': get_ref_for_pid('acac', martigny_root_account.pid)},
+    }
+    martigny_serials_account = _make_resource(
+        client, 'acac', martigny_child_serials_account_data)
+
+    saxon_root_account_data = {
+        'name': 'Saxon root account',
+        'number': 'SXN.0000.00',
+        'allocated_amount': 20000,
+        'budget': {'$ref': get_ref_for_pid('budg', budget_2020_martigny.pid)},
+        'library': {'$ref': get_ref_for_pid('lib', lib_saxon.pid)}
+    }
+    saxon_root_account = _make_resource(
+        client, 'acac', saxon_root_account_data)
+
+    saxon_child_books_account_data = {
+        'name': 'Saxon Books chid account',
+        'number': 'SXN.000b.00',
+        'allocated_amount': 2500,
+        'budget': {'$ref': get_ref_for_pid('budg', budget_2020_martigny.pid)},
+        'library': {'$ref': get_ref_for_pid('lib', lib_saxon.pid)},
+        'parent': {'$ref': get_ref_for_pid('acac', saxon_root_account.pid)},
+    }
+    saxon_books_account = _make_resource(
+        client, 'acac', saxon_child_books_account_data)
+
+    saxon_child_serials_account_data = {
+        'name': 'Saxon Serials chid account',
+        'number': 'SXN.000s.00',
+        'allocated_amount': 4000,
+        'budget': {'$ref': get_ref_for_pid('budg', budget_2020_martigny.pid)},
+        'library': {'$ref': get_ref_for_pid('lib', lib_saxon.pid)},
+        'parent': {'$ref': get_ref_for_pid('acac', saxon_root_account.pid)},
+    }
+    saxon_serials_account = _make_resource(
+        client, 'acac', saxon_child_serials_account_data)
+
+    manual_controls = [
+        {
+            '1':
+                {
+                 'name': martigny_root_account,
+                 'balance': (5000, 10000),
+                 'expenditure': (0, 0),
+                 'encumbrance': (0, 0)
+                }
+        },
```

Comment:
What a strange structure .... Why do you need a numeric key for each object of the list ? Why use a list, you could just use a dict.
```python
controls = {
    martiny_root_account: {
        'balance': (5000, 10000),
        'expenditure': (0, 0),
        'encumbrance': (0, 0)
     },
     martigny_books_account: {
     ....
 }
```
Any kind of hashable object could be used as key for a dict

Reply:
will not work `TypeError: unhashable type: 'Account`